packages/handbook-v1/en/Basic Types.md

```diff
@@ -175,42 +175,79 @@ let colorName: string = Color[2];
 console.log(colorName); // Displays 'Green' as its value is 2 above
 ```
 
-# Any
+# Unknown
 
 We may need to describe the type of variables that we do not know when we are writing an application.
-These values may come from dynamic content, e.g. from the user or a 3rd party library.
-In these cases, we want to opt-out of type checking and let the values pass through compile-time checks.
-To do so, we label these with the `any` type:
+These values may come from dynamic content &ndash; e.g. from the user &ndash; or we may want to intentionally accept all value in our API.
+In these cases, we want to provide a type that tells the compiler and future readers that this variable could be anything, so we give it the `unknown` type.
 
 ```ts twoslash
-let notSure: any = 4;
+let notSure: unknown = 4;
 notSure = "maybe a string instead";
 notSure = false; // okay, definitely a boolean
 ```
 
-The `any` type is a powerful way to work with existing JavaScript, allowing you to gradually opt-in and opt-out of type checking during compilation.
 You might expect `Object` to play a similar role, as it does in other languages.
-However, variables of type `Object` only allow you to assign any value to them. You can't call arbitrary methods on them, even ones that actually exist:
+However, variables of type `Object` do not allow you to assign `null` or `undefined` when `--strictNullChecks` is specified.
 
 ```ts twoslash
-// @errors: 2339
-let notSure: any = 4;
-notSure.ifItExists(); // okay, ifItExists might exist at runtime
-notSure.toFixed(); // okay, toFixed exists (but the compiler doesn't check)
+// @errors: 2322
+const notSure: Object = null; // error, type 'null' is not assignable to type 'Object'
+```
```

Comment:
I think this section should be moved to `any`.  I don't think there is confusion around `Object` vs `unknown` as they fulfill quite different purposes and have relatively little overlap.

Reply:
Actually, I wonder if we should get rid of it entirely since there is a lot more difference between `any` and `Object` than just the ability to assign null/undefined.

Reply:
Agree

Reply:
> I think this section should be moved to `any`. I don't think there is confusion around `Object` vs `unknown` as they fulfill quite different purposes and have relatively little overlap.

I don't know that much about the details how TypeScript handles `Object`, so I might be wrong. To me, `Object` looks much more similar to `unknown` than to `any`. For example in Java:

```java
Object str = "some string";
str.length(); // Error because length() does not exist on Object. With any, that would not be an error.
```

Anyway, I think this section doesn't have much value anymore. Previously, this section highlighting a specific difference between `Object` and `any`. This difference is gone with `unknown` and we already have a comparision between `unknown` and `any` below. I agree that removing the section is the right decision.